sdk/js/src/service/gateways.js

```diff
@@ -74,11 +75,21 @@ class Gateways {
 
   // Create
 
-  async create (userId = this._defaultUserId, gateway) {
+  async create (userId = this._defaultUserId, gateway, { setDefaults = true }) {
+    let gtw = gateway
+    const Url = URL ? URL : window.URL
+
+    if (setDefaults) {
```

Comment:
We decided to not use the any set default flag anymore in the SDK:
https://github.com/TheThingsNetwork/lorawan-stack/issues/683
The server address should rather be passed explicitly to avoid *magic*.

(It also still needs to be removed from the device service)